app/commands/user/insiders_status/update.rb

```diff
@@ -27,6 +27,7 @@ def call
     User::SetDiscordRoles.defer(user)
     User::SetDiscourseGroups.defer(user)
     User::Notification::CreateEmailOnly.defer(user, @notification_key) if @notification_key
+    User::Premium::Update.defer(user)
```

Comment:
This is the case where someone **loses** Insiders + Premium, right?

Reply:
Correct